sdk/tables/Azure.Data.Tables/src/TableServiceClient.cs

```diff
@@ -214,6 +214,42 @@ public TableServiceClient(string connectionString, TableClientOptions options =
             _secondaryServiceOperations = new ServiceRestClient(_diagnostics, _pipeline, secondaryEndpoint, _version);
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TableServiceClient"/> using the specified <see cref="Uri" />.
+        /// </summary>
+        /// <param name="endpoint">
+        /// A <see cref="Uri"/> referencing the table service account.
+        /// This is likely to be similar to "https://{account_name}.table.core.windows.net/" or "https://{account_name}.table.cosmos.azure.com/".
+        /// </param>
+        /// <param name="tokenCredential">The <see cref="TokenCredential"/> used to authorize requests.</param>
+        /// <param name="options">
+        /// Optional client options that define the transport pipeline policies for authentication, retries, etc., that are applied to every request.
+        /// </param>
```

Comment:
Does this endpoint not validated `https`?   Should we consider mirroring the `<exception>` comment here?

Reply:
We only validate for the SAS token scenario because the credential is in the URI